src/Symfony/Bundle/FrameworkBundle/HttpCache/HttpCache.php

```diff
@@ -54,7 +54,6 @@ public function __construct(HttpKernelInterface $kernel, $cacheDir = null)
     protected function forward(Request $request, $raw = false, Response $entry = null)
     {
         $this->getKernel()->boot();
-        $this->getKernel()->getContainer()->set('cache', $this); // to be removed in 4.0?
```

Comment:
perhaps move to a deprecated definition? it aliases @kernel basically. Otherwise keep it / uncomment :)